tests/testthat/test-validate-inputs.R

```diff
@@ -12,6 +12,20 @@ test_that("PJNZ can be validated and return data", {
   })
 })
 
+test_that("PJNZ can be validated and return data, in french", {
+  pjnz <- file_object(file.path("testdata", "Botswana2018.PJNZ"))
+  expect_equal(do_validate_pjnz(pjnz),
+               list(data = list(country = scalar("Botswana"),
+                                iso3 = scalar("BWA")),
+                    filters = scalar(NA)))
+  mock_read_country <- mockery::mock("GBR")
```

Comment:
This going to clash with multiple PJNZ update just done (the validation which says if GBR then fail has been removed) - I can make an equivalent test in that PR.